module2/src/com/epam/onedimensionalarrays/CalculateSumOfArray.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.onedimensionalarrays;
+
+public class CalculateSumOfArray {
+    public static int calculateSumOfArray(int[] array, int divider) {
```

Comment:
make the name of method more readable
```suggestion
    public static int countSum(int[] array, int divider) {
```